packages/core/foundry.toml

```diff
@@ -0,0 +1,7 @@
+[profile.default]
+allow_paths = ["../../node_modules","/../../node_modules","../node_modules"]
```

Comment:
What is `"/../../node_modules"`? Is this a nonstandard file path or something? Otherwise, it looks like an alias for `/node_modules`, which would be an odd place to look for imports...?

Also, would `"../node_modules"` be `protocol/packages/node_modules`? Unless the repo is restructured, there shouldn't be any node_modules folder there, so we might be able to remove that one.

Reply:
I actually dont think we need all of these, going to minimize now.

Reply:
Awesome, +1